global.html

```diff
@@ -99,6 +103,9 @@
         };
         xhr.send();
       }
+
+      safari.extension.addContentScriptFromURL(safari.extension.baseURI + 'inject.js', ['safari-reader://*/*'], null, true);
+      safari.extension.addContentStyleSheetFromURL(safari.extension.baseURI + 'style.css', ['safari-reader://*/*'], null, true);
```

Comment:
yeah, I tried the same thing using just the whitelist and couldn't get it to work.